### ISSUE TITLE
Identically-named playlists overwrite each other in the Export All output

### ISSUE THREAD

**Issue by pavelkomarov** [pavelkomarov/exportify#12]
Discovered by @shape55 over on #9. I need [a way to see whether a file already exists in the JSZip](https://stuk.github.io/jszip/documentation/api_jszip/file_name.html) and add `_#` to the end of the filename if it does (or maybe prepend the name of the folder where the playlist lives?), because JSZip's [`file(name, data)` function also serves a value-updating purpose](https://stuk.github.io/jszip/documentation/api_jszip/file_data.html).

I can't see where in @watsonbox's code this could have been addressed. He's accumulating names by calling the `fileName` function, then using those as keys to load values in to the zip, just like I am. It's possible this issue went completely unnoticed because anyone with few enough playlists to successfully use the not-rate-limited Export All functionality [over there](https://rawgit.com/watsonbox/exportify/master/exportify.html) is unlikely to have any with duplicate names.

**Comment by pavelkomarov** [comment]
Okay, it works. Just tried it with my own dummy duplicates. And as a bonus I made it so the errors displayed at the top of the page don't overwrite each other on an Export All, so you should be able to see exactly how many failed and should be missing from the zip.

**Comment by shape55** [comment]
Ok, awesome, on both counts.  I'm running Export All again now and will report back when it's done.

Also, I think I had the wrong Console open before in Firefox: the Web Console rather than the Browser Console.  The Web Console has the "Persists logs" checkbox at the top-right that I was referring to in that [#9 comment](https://github.com/pavelkomarov/exportify/issues/9#issuecomment-517486989).

**Comment by shape55** [comment]
It works! :^)  The zip file contains 473 of 477 playlists, and the missing 4 are all listed at the top.  So all the playlists w/ duplicate names are being renamed correctly now.  Thanks so much for putting the time into making a working version of this app.  I'll post a followup comment on the other forks I tried w/ a link to your repository.

Lastly, any idea why those 2 playlists w/ undefined errors I listed [here](https://github.com/pavelkomarov/exportify/issues/10#issuecomment-517494042) cause your fork to crash but not watsonbox's?
